jupyter/Scale Output Example.ipynb

```diff
@@ -0,0 +1,182 @@
+{
```

Comment:
This file is called "Scale Output Example.ipynb" when it should probably be "Scalar" instead.

Reply:
Done